### PR TITLE
Full size SVG - works better for mobile since to horizontal scroll (pinch better too)

### PR DESCRIPTION
**Pull request by jread-usgs**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

Similar to the other open PRs, I didn't lint because it would create a bunch of diffs that aren't related to this work. This fix is for the width of the carto SVG mentioned [here](https://github.com/usgs-makerspace/makerspace-sandbox/issues/608). 

image

### CODE REVIEW

File: src/components/MapImageSlider.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
         id="cartogram-svg"
         xmlns="http://www.w3.org/2000/svg"
         xmlns:xlink="http://www.w3.org/1999/xlink"
-        viewBox="0 -50 770 420"
+        viewBox="0 -50 502 405"
```
Comment (lindsayplatt) [inline src/components/MapImageSlider.vue:17]: when we rebuild the SVG, this change will get wiped out

Comment (lindsayplatt) [reply src/components/MapImageSlider.vue:17]: Can you fix here, too? https://github.com/usgs-makerspace/gages-through-the-ages/blob/master/data_processing_pipeline/6_visualize.yml#L23-L26

Comment (lindsayplatt) [reply src/components/MapImageSlider.vue:17]: I don't think you need to rebuild it, but just so that the next time someone does, it is fixed.

Comment (jread-usgs) [reply src/components/MapImageSlider.vue:17]: updated

**Pull request merged**